### PR TITLE
Generate Invite Link

### PR DESCRIPTION
**Pull request by LuisanSuarez**

### CODE REVIEW

File: endpoints.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    .get(`${url}/api/group/${this.state.userID}`)
+      .then(response => {
+      this.setState({ groups: response.data}))
+      console.log(response);
+      console.log(response.data);
+    })
+    .catch(err =>{
+      console.log('error',err);
+    })
+  }
```
Comment (PhilHart23) [inline endpoints.js:20]: This one looks good too!

**Review by PhilHart23 — APPROVED** [review]

File: client/src/components/InviteGenerator.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+const url = "http://chore-monkey.herokuapp.com"
+
+
+class InviteGenerator extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            inivteLink: "",
+            groupID: 1,   //UPDATE THIS: we need to know where to fetch the group ID's from
```
Comment (secantsquared) [inline client/src/components/InviteGenerator.js:12]: agreed.

File: endpoints.js (hunk 1 above)
Comment (secantsquared) [inline endpoints.js:20]: 👍

**Review by secantsquared — APPROVED** [review]
✔️

**Pull request merged**